jerry-core/ecma/builtin-objects/ecma-builtin-array-prototype.c

```diff
@@ -981,8 +981,10 @@ ecma_builtin_array_prototype_object_slice (ecma_value_t arg1, /**< start */
 static ecma_value_t
 ecma_builtin_array_prototype_object_sort_compare_helper (ecma_value_t lhs, /**< left value */
                                                          ecma_value_t rhs, /**< right value */
+                                                         ecma_object_t *array_buffer_p, /**< arrayBuffer */
```

Comment:
I would put this after `compare_func`. It is an extra paramter rather than a core. Change in all places.